school_lib.rb

```diff
@@ -0,0 +1,8 @@
+require './lib/app'
+
+def main
+  app = App.new
+  app.main_menu
+end
+
+main
```

Comment:
- [ ] Since this is the `main.rb` file, it should do the following as per requirement:

- Present the user with a list of options to perform.
- Let users choose an option.
- If needed, ask for parameters for the option.
- Have a way to quit the app.

Currently, this is being handled in `app.rb`

Main shouldlokk something like the following:


```
def main
  app = App.new()

  response = nil
 
  # *Present the user with a list of options to perform.
  puts "Welcome to School Library App!\n\n"
  while response != "7"
    puts "Please choose an option by enterin a number:"
    puts "1 - List all books"
    puts "2 - List all people"
    puts "3 - Create a person"
    puts "4 - Create a book"
    puts "5 - Create a rental"
    puts "6 - List all rentals for a given person id"
    puts "7 - Exit"
    # *Lets users choose an option.
    response = gets.chomp

    case response
    when "1"
      app.list_books()
    when "2"
      app.list_people()
    when "3"
      app.create_person()
    when "4"
      app.create_book()
    when "5"
      app.create_rental()
    when "6"
      app.list_rentals_for_person_id()
    when "7"
      # *Have a way to quit the app.
      puts "Thank you for using this app!"
    end

    puts "\n"
  end
end
main()

Reply:
@Munsa1 thank for the tips